lib/spaceship/tunes/tunes_client.rb

```diff
@@ -596,6 +596,7 @@ def submit_testflight_build_for_review!(app_id: nil, train: nil, build_number: n
         current['privacyPolicyUrl']['value'] = privacy_policy_url
         current['pageLanguageValue'] = current['language'] # There is no valid reason why we need this, only iTC being iTC
       end
+      build_info['significantChange'] ||= { value: '' }
```

Comment:
Ah, my ruby-fu is weak, thanks for the heads-up @snatchev. @KrauseFx, I guess
```ruby
 build_info['significantChange'] ||= {}
```
should be enough, right? I'll make an update.